modules/account-lib/src/coin/xrp/keyPair.ts

```diff
@@ -9,5 +13,115 @@ export class KeyPair extends Secp256k1ExtendedKeyPair {
    */
   constructor(source?: KeyPairOptions) {
     super(source);
+    if (!source) {
+      this.hdNode = HDNode.fromSeedBuffer(randomBytes(512 / 8));
```

Comment:
What's the reasoning behind the numbers `randomBytes(512 / 8)`

Reply:
I have changed this to 64 bytes. We have chosen the 64 bytes as the master seed size because bip32 recommends a seed greater than 32 bytes at least.